assets/_sass/bootstrap/bootstrap/_tables.scss

```diff
@@ -195,7 +194,7 @@ table {
         > tr {
           > th,
           > td {
-            white-space: nowrap;
+            white-space: normal;
```

Comment:
I'm still getting the same issue and we shouldn't be changing the bootstrap styles
![Screenshot 2022-07-26 at 12 43 13](https://user-images.githubusercontent.com/48946187/180987991-9906d933-9dd5-4281-9071-87c9f45dfa64.png)

I will send you a copy of the Apple Handbook to test with

Reply:
Sorry, I missed your other [PR](https://github.com/SPANDigital/presidium-theme-apple/pull/24). I see it's working now but I still think we need to override the bootstrap styles instead of changing them directly.